tests/integration/ambient/main_test.go

```diff
@@ -93,7 +93,6 @@ func TestMain(m *testing.M) {
 	framework.
 		NewSuite(m).
 		RequireMinVersion(24).
-		Label(label.IPv4). // https://github.com/istio/istio/issues/41008
```

Comment:
Do we have to set this based on the ipfamily of the cluster or skip completely?

Reply:
The label is meant to be "Do not run this test if its in IPv6". Given ambient should run on v4 or v6 clusters, we should not have any skip